<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>asset_register_Form</class>
 <widget class="QWidget" name="asset_register_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="category_horizontalLayout">
     <item>
      <widget class="QLabel" name="category_label">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="category_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="category_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="new_asset_pushButton">
       <property name="text">
        <string>Register New Asset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="main_horizontalLayout" stretch="1,4">
     <item>
      <layout class="QVBoxLayout" name="asset_list_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="filter_horizontalLayout">
         <item>
          <widget class="QLabel" name="filter_list_label">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="filter_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="filter_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="asset_listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="options_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="asset_name_horizontalLayout">
         <item>
          <widget class="QLabel" name="asset_name_label">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Asset Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="asset_name_lineEdit">
           <property name="toolTip">
            <string>Enter a name for this asset</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="asset_namespace_label">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Asset Namespace:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="asset_namespace_lineEdit">
           <property name="toolTip">
            <string>Enter an optional override namespace for this asset.</string>
           </property>
           <property name="placeholderText">
            <string>optional namespace name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="asset_subtype_horizontalLayout">
         <item>
          <widget class="QLabel" name="asset_subtype_label">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Asset Subtype:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="asset_subtype_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="asset_subtype_comboBox">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="asset_path_horizontalLayout">
         <item>
          <widget class="QLabel" name="asset_path_label">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Asset Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="asset_path_lineEdit">
           <property name="toolTip">
            <string>Path to asset folder</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="file_browser_pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>83</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Select Path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="archetype_horizontalLayout">
         <item>
          <widget class="QLabel" name="archetype_label">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Archetype</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="archetype_lineEdit">
           <property name="toolTip">
            <string>Namespace for rig</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
